qutebrowser/browser/qutescheme.py

```diff
@@ -201,6 +203,28 @@ def qute_bookmarks(_url):
     return 'text/html', html
 
 
+@add_handler('tabs')
+def qute_tabs(_url):
+    """Handler for qute://tabs. Display information about all open tabs."""
+    tabs = collections.defaultdict(list)
+    for win_id, window in objreg.window_registry.items():
+        if sip.isdeleted(window):
+            continue
+        win_id_str = str(win_id)
+        tabbed_browser = objreg.get('tabbed-browser',
+                                    scope='window',
+                                    window=win_id)
+        for tab_idx in range(tabbed_browser.count()):
+            tab_fields = tabbed_browser.get_tab_fields(tab_idx)
+            tabs[win_id_str].append(
+                (tab_fields['title'], tab_fields['current_url']))
```

Comment:
I didn't notice this before, but you should be able to simplify this by using the actual tab objects instead of indices and `get_tab_fields`. Something like (untested):

```python
for tab in tabbed_browser.widgets():
    tabs[win_id_str].append((tab.title(), tab.url()))
```